src/containers/Network/types.ts

```diff
@@ -39,3 +39,52 @@ export interface NodeData extends NodeResponse {
   // eslint-disable-next-line camelcase -- mimicking rippled
   load_factor: number | null
 }
+
+export interface ValidatorResponse {
+  // eslint-disable-next-line camelcase -- from VHS
+  validation_public_key: string
+  // eslint-disable-next-line camelcase -- from VHS
+  signing_key: string
+  // eslint-disable-next-line camelcase -- from VHS
+  master_key?: string
+  revoked?: boolean
+  domain: string
+  chain: string
+  networks?: string
+  // eslint-disable-next-line camelcase -- from VHS
+  current_index: number
+  // eslint-disable-next-line camelcase -- from VHS
+  server_version?: string
+  // eslint-disable-next-line camelcase -- from VHS
+  agreement_1h: {
```

Comment:
This could be its own type like 

```

export interface ValidatorScore {
    missed: number
    total: number
    score: string
    incomplete: boolean
}